libs/blocks/merch/merch.js

```diff
@@ -100,6 +100,14 @@ window.tacocat.loadPromise = new Promise((resolve) => {
 
   loadScript(literalScriptUrl)
     .catch(() => ({})) /* ignore if literals fail */
+    .then(() => {
+      // load custom elements polyfill for Safari only.
+      if (navigator.userAgent.indexOf('Safari') >= 0 && navigator.userAgent.indexOf('Chrome') < 0) {
+        const base = miloLibs || codeRoot;
+        return loadScript(`${base}/deps/custom-elements-1.3.0.js`);
+      }
+      return null;
```

Comment:
Should this check live here or within the tacocat script?

Reply:
we will move and merge tacocat back in milo commerce as per https://git.corp.adobe.com/wcms/team/discussions/9.

We might need to shift things around a bit more, but for now this is to fix the immediate issue with Safari.